recipe-server/client/control_new/index.js

```diff
@@ -20,33 +19,40 @@ const reducers = combineReducers({
   router: routerReducer,
 });
 
+const middleware = [
+  routerMiddleware,
+  thunk,
+];
 
-const store = createStore(reducers, reducers(undefined, { type: 'initial' }), compose(
-  applyMiddleware(
-    routerMiddleware,
-    thunk,
+if (DEVELOPMENT) {
+  middleware.push(
     createLogger({
```

Comment:
I comment out the `createLogger` business pretty consistently while working on things; it really hits performance on a page that has a lot of redux actions (listings, history timeline). If you don't have a particular use for it, I'd vote to take out `redux-logger` all together.